lib/versioning/loose/utils.spec.ts

```diff
@@ -34,8 +34,15 @@ describe('versioning/loose/utils', () => {
         return _version ? _version.localeCompare(_other) : 0;
       }
 
-      protected _parse(_version: string): GenericVersion {
-        return _version === 'test' ? null : { release: [1, 0, 0] };
+      protected _parse(_version: string): GenericVersion | null {
+        const matchGroups = _version.match(
+          /^(?<major>\d)\.(?<minor>\d)\.(?<patch>\d)$/
+        )?.groups;
+        if (!matchGroups) {
+          return null;
+        }
+        const { major, minor, patch } = matchGroups;
+        return { release: [major, minor, patch].map(Number) };
```

Comment:
```suggestion
        return { release: [major, minor, patch].map(n => parseInt(n, 10)) };
```
I think this is more safe